tools/gfx/vulkan/render-vk.cpp

```diff
@@ -38,14 +38,14 @@
 namespace gfx {
 using namespace Slang;
 
-class VKDevice : public GraphicsAPIRenderer
+class VKDevice : public RendererBase
 {
 public:
     enum
     {
         kMaxRenderTargets = 8,
         kMaxAttachments = kMaxRenderTargets + 1,
-
+        kMaxPushConstantSize = 256,
         kMaxDescriptorSets = 8,
```

Comment:
We eventually need to query these via VK API calls and be ready for them to be different across implementations.